mantidproject/mantid
# OSIRISDiffractionReduction fixes

Fixes #13015.

(have page 32 of the [OSIRIS manual](http://www.isis.stfc.ac.uk/instruments/osiris/documents/osiris-user-guide6672.pdf) open when testing this, note that when a Mantid uses the same dRanges as Ariel)

To test:
- Open Indirect > Diffraction
- Instrument: OSIRIS, diffonly
- Run with the following setups:
- dRange detection fails:
  - Sample: 10200
  - Vanadium: 10153
  - Manual dRange: disabled
  - In the logs you will see the algorithm detects dRange 3 (3.1-4.3 Angstrom), this run and vanadium are actually for dRange 1 (note the time channel overlap)
- Manual dRange:
  - Sample: 10200
  - Vanadium: 10153
  - Manual dRange: enabled
  - dRange: 1

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24396&oldid=24352).
